kits/release-notes-generator/agent.md

```
@@ -0,0 +1,54 @@
+| Field | Type | Required | Description |
+|---|---|---|---|
+| `changes` | `string` | Yes | Newline-separated commit messages and/or PR titles. |
+
+## Outputs
+| Field | Type | Description |
+|---|---|---|
+| `releaseNotes` | `string` | Categorized release notes in Markdown. |
+
+## Example
+**Input (`changes`):**
+```text
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)